operator/controllers/common/taskdefinitionhandler.go

```diff
@@ -0,0 +1,70 @@
+package common
+
+import (
+	"os"
+	"reflect"
+
+	klcv1alpha3 "github.com/keptn/lifecycle-toolkit/operator/apis/lifecycle/v1alpha3"
+)
+
+const (
+	FunctionRuntimeImageKey = "FUNCTION_RUNNER_IMAGE"
+	PythonRuntimeImageKey   = "PYTHON_RUNNER_IMAGE"
+	FunctionScriptMountPath = "/var/data/function.ts"
+	PythonScriptMountPath   = "/var/data/function.py"
+)
+
+func GetRuntimeSpec(def *klcv1alpha3.KeptnTaskDefinition) *klcv1alpha3.RuntimeSpec {
+
+	if !IsRuntimeEmpty(def.Spec.Function) {
+		return def.Spec.Function
+	}
+	if !IsRuntimeEmpty(def.Spec.Deno) {
+		return def.Spec.Deno
+	}
+	if !IsRuntimeEmpty(def.Spec.Python) {
+		return def.Spec.Python
+	}
+
+	return nil
+}
+
+func IsRuntimeEmpty(spec *klcv1alpha3.RuntimeSpec) bool {
+	return spec == nil || reflect.DeepEqual(spec, &klcv1alpha3.RuntimeSpec{})
+}
+
+func IsVolumeMountPresent(spec *klcv1alpha3.ContainerSpec) bool {
+	return spec != nil && spec.Container != nil && spec.VolumeMounts != nil && len(spec.VolumeMounts) > 0
+}
+
+func IsInline(spec *klcv1alpha3.RuntimeSpec) bool {
+	return spec != nil && !reflect.DeepEqual(spec.Inline, klcv1alpha3.Inline{})
+}
+
+func GetRuntimeImage(def *klcv1alpha3.KeptnTaskDefinition) string {
+	image := os.Getenv(FunctionRuntimeImageKey)
+	if !IsRuntimeEmpty(def.Spec.Python) && IsRuntimeEmpty(def.Spec.Function) {
+		return os.Getenv(PythonRuntimeImageKey)
```

Comment:
nitpick, to stay consistent with the `GetRuntimeMountPath` function below
```suggestion
		image = os.Getenv(PythonRuntimeImageKey)
```